src/utils/date-time.js

```diff
@@ -129,6 +147,45 @@ export default {
     return m + '/' + d + '/' + y;
   },
 
+  /**
+   * formatTime, extracted from date-picker/date-picker.
+   *
+   * @param date [Date] A Date object.
+   * @param format [String] One of 'ampm', '24hr', defaults to 'ampm'.
+   * @param pedantic [Boolean] Check time-picker/time-picker.jsx file.
+   *
+   * @return String A string representing the formatted time.
+   */
+  formatTime(date, format = 'ampm', pedantic = false) {
+    let hours = date.getHours();
```

Comment:
Throwing `Uncaught TypeError: Cannot read property 'getHours' of null` on the `/#/components/time-picker` page of the documentation.